src/algebra/monoid_algebra.lean

```diff
@@ -795,14 +861,15 @@ to definitional inconveniences, we can still show the types are isomorphic.
 -/
 
 /-- The equivalence between `add_monoid_algebra` and `monoid_algebra` in terms of
-`multiplicative` -/
-protected def add_monoid_algebra.to_multiplicative [semiring k] [add_monoid G] :
+`multiplicative`
+protected def add_monoid_algebra.to_multiplicative [semiring k] [has_add G] :
   add_monoid_algebra k G ≃+* monoid_algebra k (multiplicative G) :=
-{ map_mul' := λ x y, by convert add_monoid_algebra.map_domain_mul (add_hom.id G),
+{ map_mul' := λ x y, by convert monoid_algebra.map_domain_mul (mul_hom.id (multiplicative G)),
   ..finsupp.dom_congr multiplicative.of_add }
+-/
```

Comment:
My hunch is that this change broke leanchecker ([zulip thread](https://leanprover.zulipchat.com/#narrow/stream/144837-PR-reviews/topic/.236572.20crashes.20leanchecker/near/229213749))